query.go

```diff
@@ -519,20 +543,10 @@ func queryChainServiceBatch(
 				}
 			}
 		case <-time.After(qo.timeout):
-			// Check if we're done; if so, quit.
```

Comment:
So the timeout send it just a noop now?

Reply:
It already was if we weren’t done with all the queries, so it’d just wait again.  There’s no need to check if we’re done anymore since it’s handled when receiving responses.

Reply:
Hmm so then timeouts haven't worked at all this whole time.